app/views/users/registrations/new.html.erb

```diff
@@ -6,7 +6,7 @@
     <h2 class="mt-6 text-center text-3xl font-bold tracking-tight text-gray-900 dark:text-gray-200">Sign up for free</h2>
     <p class="block mt-2 text-center text-sm text-gray-600 dark:text-gray-300">
       Or
-      <%= link_to 'sign in to your existing account', sign_in_path, class: 'font-medium text-gold-600 hover:text-gold-500' %>
+      <%= link_to 'sign in to your existing account', sign_in_path, class: 'font-medium underline hover:no-underline text-gray-600 hover:text-gray-800 dark:text-gray-400 dark:hover:text-gray-300' %>
```

Comment:
Nit: This may fades into the background a bit too much imo. We could stand to make it more of a cta:

```suggestion
      <%= link_to 'sign in to your existing account', sign_in_path, class: 'font-medium underline hover:no-underline text-gray-700 hover:text-gray-800 dark:text-gray-300 dark:hover:text-gray-300' %>
```